pkgs/applications/video/go-chromecast/default.nix

```diff
@@ -0,0 +1,21 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "go-chromecast";
+  version = "0.0.11";
+
+  src = fetchFromGitHub {
+    owner = "vishen";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

works as well.

Reply:
FWIW if upstream has a /branch/ by this name ("v0.0.11") this may do the wrong thing.  Not very likely here, but I've seen this cause breakage elsewhere which is why I started specifying `refs/tags/TAG` explicitly.  I'm happy with the change, but they're not quite the same :3.

Reply:
Oh I see, good point! Never mind the suggested change then.